packaging/docker/README.md

```diff
@@ -16,6 +16,18 @@ Further documentation can be found at http://docs.grafana.org/installation/docke
 
 ## Changelog
 
+### v6.3.0-beta1
```

Comment:
I think we don't update this anymore and use the changelog.md in repo root, but not sure.

Reply:
It's probably not a bad idea to keep this updated anyway since we are keeping this document manually updated on Docker Hub.